File: phong_tap_fe/quan-ly-phong-gym/src/common/shared/enums/hinhThucThanhToan.enums.ts
export enum HinhThucThanhToanEnums {
  THANHTOANCHUYENKHOANMOMO = 1,
  THANHTOANCHUYENKHOANVNPAY = 2,
  THANHTOANTIENMAT = 3,
}

export const hinhThucThanhToan: { value: number; label: string }[] = [
  {
    value: HinhThucThanhToanEnums.THANHTOANCHUYENKHOANMOMO,
    label: 'Thanh toán chuyển khoản MoMo',
  },
  {
    value: HinhThucThanhToanEnums.THANHTOANCHUYENKHOANVNPAY,
    label: 'Thanh toán chuyển khoản VNPAY',
  },
  {
    value: HinhThucThanhToanEnums.THANHTOANTIENMAT,
    label: 'Thanh toán tiền mặt',
  },
];

File: phong-tap-be/phong_tap-be/QuanLyPhongGym/src/main/java/com/example/QuanLyPhongGym/domain/enums/HinhThucThanhToanEnums.java
package com.example.QuanLyPhongGym.domain.enums;

import java.util.Map;

public enum HinhThucThanhToanEnums {
    THANHTOANCHUYENKHOANMOMO(1, "Thanh toán chuyển khoản MoMo"),
    THANHTOANCHUYENKHOANVNPAY(2, "Thanh toán chuyển khoản VNPAY"),
    THANHTOANTIENMAT(3, "Thanh toán tiền mặt");

    public static final Map<Integer, HinhThucThanhToanEnums> dict = Map.of(
            HinhThucThanhToanEnums.THANHTOANCHUYENKHOANMOMO.value, HinhThucThanhToanEnums.THANHTOANCHUYENKHOANMOMO,
            HinhThucThanhToanEnums.THANHTOANCHUYENKHOANVNPAY.value, HinhThucThanhToanEnums.THANHTOANCHUYENKHOANVNPAY,
            HinhThucThanhToanEnums.THANHTOANTIENMAT.value, HinhThucThanhToanEnums.THANHTOANTIENMAT);

    public final Integer value;
    public final String label;

    HinhThucThanhToanEnums(Integer value, String label) {
        this.value = value;
        this.label = label;
    }
}
